lib/play.rb

```diff
@@ -0,0 +1,3 @@
+require_relative 'blackjack_rb.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.